src/Microsoft.AspNetCore.CookiePolicy/SameSitePolicy.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNetCore.CookiePolicy
+{
+    public enum MinimumSameSiteStrictnessPolicy
```

Comment:
MinimumSameSitePolicy?
Since the values match the underlying enum, do we actually need this one?